web/models.py

```
@@ -2578,6 +2578,97 @@
+class PDFType(models.Model):
+    """Model for categorizing different types of PDF submissions."""
+
+    name = models.CharField(max_length=100)
+    description = models.TextField(blank=True)
+    icon_class = models.CharField(max_length=50, blank=True, help_text="Font Awesome icon class")
+
+    class Meta:
+        verbose_name = "PDF Type"
+        verbose_name_plural = "PDF Types"
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)